src/ds/ds6/ds6-private.h

```diff
@@ -53,5 +53,113 @@ namespace librealsense
 
         bool ds6_try_fetch_usb_device(std::vector<platform::usb_device_info>& devices,
             const platform::uvc_device_info& info, platform::usb_device_info& result);
+
+        enum class ds6_calibration_table_id
+        {
+            coefficients_table_id = 0xb4, // TODO - move to ds6
```

Comment:
`TODO - move to ds6`, it's already here right?

Reply:
done